<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1446</width>
    <height>969</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>맑은 고딕</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
				<property name="text">
					<string>옷 이름 :</string>
				</property>
			</widget>
			<widget class="QLineEdit" name="lineEdit_third_clothname">
				<property name="geometry">
					<rect>
						<x>130</x>
						<y>110</y>
						<width>113</width>
						<height>25</height>
					</rect>
				</property>
			</widget>
			<widget class="QPushButton" name="pushButton">
				<property name="geometry">
					<rect>
						<x>260</x>
						<y>100</y>
						<width>112</width>
						<height>34</height>
					</rect>
				</property>
				<property name="text">
					<string>PushButton</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_prices">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>200</y>
						<width>71</width>
						<height>18</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>굴림</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
			</widget>
			<widget class="QLabel" name="label_average_min">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>340</y>
						<width>211</width>
						<height>61</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>굴림</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
				<property name="alignment">
					<set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
				</property>
			</widget>
			<widget class="QLabel" name="label_empty">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>180</y>
						<width>171</width>
						<height>16</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>굴림</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
			</widget>
			<widget class="QLabel" name="label_6">
				<property name="geometry">
					<rect>
						<x>160</x>
						<y>180</y>
						<width>211</width>
						<height>141</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>굴림</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
				<property name="alignment">
					<set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
				</property>
			</widget>
			<widget class="QLabel" name="label_averageprice">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>390</y>
						<width>211</width>
						<height>141</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>굴림</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
				<property name="alignment">
					<set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
				</property>
			</widget>
			<widget class="QLabel" name="label_recommend_color">
				<property name="geometry">
					<rect>
						<x>510</x>
						<y>100</y>
						<width>451</width>
						<height>361</height>
					</rect>
				</property>
				<property name="font">
					<font>
						<family>Agency FB</family>
						<pointsize>12</pointsize>
					</font>
				</property>
				<property name="text">
					<string/>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>1025</width>
					<height>21</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
